statics/js/components/topology.js

```diff
@@ -358,7 +358,7 @@ var TopologyComponent = {
 
     currentNodeFlowsQuery: function() {
       if (this.currentNode && this.currentNode.isCaptureAllowed())
-        return "G.V('" + this.currentNode.id + "').Flows().Dedup().Sort()";
+        return "G.Flows().Has('NodeTID', '" + this.currentNode.metadata.TID + "').Sort()";
```

Comment:
Is it still required as you changed the "V().Flows()" behaviour ?

Reply:
I think it still better to do it, as internally it will do the same, except that it will have to lookup for the node while here we already have it.